plenum/test/node_catchup/test_discard_view_no.py

```diff
@@ -43,8 +43,9 @@ def testNodeDiscardMessageFromUnknownView(txnPoolNodeSet,
     # 3 pc msg (PrePrepare) needs to be discarded
     _, did = sdk_wallet_client
     primaryRepl = getPrimaryReplica(txnPoolNodeSet)
+    inst_id = 0
     three_pc = PrePrepare(
```

Comment:
Can we use a helper method `create_pre_prepare` for creation of PrePrepare?

Reply:
Changed.